helper/inheritance-resolver.js

```diff
@@ -11,8 +11,17 @@ module.exports = function(plugins, config, name) { // eslint-disable-line func-n
   }
 
   function generateSymlinks(src, dest, replacePattern, ignore = []) {
+    var themeIgnoreFiles = [];
+    // Iterate through the config.themes to grab any ignored paths that
+    // themers want to be excluded from the gulp.src() and sass compilation
+    Object.keys(config.themes).forEach(name => {
```

Comment:
I don't like this part. Theme specific configuration shouldn't be used for all themes.